metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -221,7 +221,7 @@ class MetalsLanguageServer(
       params => didChangeWatchedFiles(params),
     )
   )
-  private val indexingPromise: Promise[Unit] = Promise[Unit]()
+  val indexingPromise: Promise[Unit] = Promise[Unit]()
```

Comment:
is this change needed?

Reply:
That's needed for the test unfortunately